ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -105,85 +111,88 @@ const CgnMerchantDetailScreen: React.FunctionComponent<Props> = (
       }
       contextualHelp={emptyContextualHelp}
     >
-      <SafeAreaView style={IOStyles.flex}>
-        {isReady(merchantDetail) ? (
-          <>
-            <ScrollView style={IOStyles.flex} bounces={false}>
-              {merchantDetail.value.imageUrl && (
-                <View style={{ paddingHorizontal: 16 }}>
-                  <Image
-                    source={{ uri: merchantDetail.value.imageUrl }}
-                    style={styles.merchantImage}
+      {isReady(merchantDetail) ? (
+        <ScrollView
+          contentContainerStyle={[
+            styles.scrollViewContainer,
+            { paddingBottom: insets.bottom }
+          ]}
+          bounces={true}
```

Comment:
```suggestion
        <ScrollView
          contentContainerStyle={[
            styles.scrollViewContainer,
            { paddingBottom: insets.bottom }
          ]}
          scrollIndicatorInsets={{ right: 1 }}
          bounces={true}
```
_Hello darkness, my old friend_

Just the same old problem with `ScrollView` scroll indicator. 😁

<img src="https://github.com/pagopa/io-app/assets/6160324/c327e081-f2c2-4685-9658-d5fbcf957417" width="200" />

Reply:
@mastro993 Here's the error I get while I'm testing this page: 
```
 ERROR  VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead. 
```
This is something we've also fixed in `IdpSelectionScreen`, too (see https://github.com/pagopa/io-app/pull/4796).
Using `scrollIndicatorInsets={{ right: 1 }}` can be a quick and easy hack, but it doesn't solve the underlying issue.

Reply:
would a `map` rendering do the job here? I can't seem to replicate the scrollbar error 😓 
example:
```js
{merchantDetail.value.discounts.map(
    (discount, index) => (
              <DiscountListItem item={discount} key={index} />
    )
  )
}
```

Reply:
I think it will be enough to set `scrollEnabled={false}` on FlatLists inside the ScrollView

Reply:
well, this is embarassing 😅 
in the meantime, I have moved the component to map functions and did a small refactor on the screen in order to remove the `connect` syntax in favor of hooks.
the errors do not show anymore and the rendering seems as expected to me, feel free to check; also, if the map syntax is an anti-pattern I'll be quick to fix 👍

Reply:
> I think it will be enough to set scrollEnabled={false} on FlatLists inside the ScrollView

@mastro993 Sadly, this is not enough to avoid that error in console. Can't understand why, though 🫠

Reply:
Problem still persists, we need `scrollIndicatorInsets={{ right: 1 }}` to fix this. Was not depending on the error shown in the console, sadly.

https://github.com/pagopa/io-app/assets/6160324/5e1c9659-0e39-48db-aedd-e594fff8cd10

Reply:
I cannot seem to reproduce it, but I have added the scrollIndicatorInsets nonetheless 👍